lib/perl/Genome/Config/AnalysisProject/Command/CompareInstrumentData.pm

```diff
@@ -0,0 +1,70 @@
+package Genome::Config::AnalysisProject::Command::CompareInstrumentData;
+
+use strict;
+use warnings;
+
+use Genome;
+
+use Set::Scalar;
+
+class Genome::Config::AnalysisProject::Command::CompareInstrumentData {
+    is => ['Command::V2'],
+    has => [
+        analysis_project_a => {
```

Comment:
It might be nice if the two parameter names differed at the beginning.  For instance, `--first-analysis-project` and `--second-analysis-project` can be shortened to `--first` and `--second`.  The current names require the entire parameter to be spelled out each time.